test_scripts/collect_data.py

```diff
@@ -0,0 +1,52 @@
+import os
+from time import sleep
+import subprocess
+import re
+
+run_time = 10
+reset_time = 5
+pps_sample = [10000,50000,100000]
+pcycle_sample = [20,200,2000]
```

Comment:
Suppose our CPU freq = 1.7 GHz. Then, 1 us = 1700 CPU cycles.
Here, 20 and 200 are too small.

Reply:
This is 1us per packet. Thus at 2000 cycles per packet we see an extra ~75us of time consumed by the packets. Since the batch size is 64 I think this makes sense. I have changed the script to test for higher cycles per packet as well.